cmds/contest/server/server.go

```diff
@@ -173,7 +244,7 @@ func ServerMain(cmd string, args []string, sigs <-chan os.Signal) error {
 	plugins.Init(pluginRegistry, ctx.Logger())
 
 	// spawn JobManager
-	listener := httplistener.NewHTTPListener(*flagListenAddr)
+	listener := httplistener.New(*flagListenAddr)
```

Comment:
Should (maybe in a future version) the listener plugin be selectable in the generator as well?

Reply:
yes, see my comment above. I am leaving it out for now as we don't have an interface for listeners yet, but I plan on adding it after this PR, and to adjust the generator to include that. But I wouldn't pack all of this in a single PR

Reply:
I'm going to resolve this